packages/typescript-vfs/test/index.test.ts

```diff
@@ -41,6 +44,27 @@ it("runs a virtual environment and gets the right results from the LSP", () => {
   `)
 })
 
+it("runs can use a FS backed system", () => {
+  const compilerOpts: ts.CompilerOptions = { target: ts.ScriptTarget.ES2016, esModuleInterop: true }
+  const fsMap = createDefaultMapFromNodeModules(compilerOpts)
+  addAllFilesFromFolder(fsMap, path.resolve(path.join(__dirname, "../../../node_modules/@types")))
+
+  const content = `import * as path from 'path';\npath.`
+  fsMap.set("index.ts", content)
+
+  const system = createFSBackedSystem(fsMap)
+  const env = createVirtualTypeScriptEnvironment(
+    system,
+    ["/node_modules/@types/node/path.d.ts", "index.ts"],
```

Comment:
@orta I really want to be able to not support the `@types` file. IDK if this is possible with how the languageService works. It seems you would need to import every file manually, gather it's imports, and then tell the language service to use those files.

Reply:
>  It seems you would need to import every file manually, gather its imports, and then tell the language service to use those files. 

So i've done this now locally but it doesn't work with how typescript handles the `@node/types`